packages/corvid-local-server/test/it/editMode.spec.js

```diff
@@ -127,11 +127,18 @@ describe("edit mode", () => {
   });
 
   it("should send updated site document when user changes page content from the editor and clicks save", async () => {
+    const elementsMap = {
+      comp1: sc.randomSdkType(),
```

Comment:
@nemni8 	i find it a bit weird exposing "randomSdkType". we can either choose specific types for the tests or generate a full "random" page content. wdyt ?